tests/python/gaia-ui-tests/gaiatest/apps/messages/regions/report.py

```diff
@@ -0,0 +1,38 @@
+import time
+
+from marionette_driver import expected, By, Wait
+from gaiatest.apps.base import Base
+from gaiatest.form_controls.header import GaiaHeader
+
+class Report(Base):
+
+    _time_sent_locator = (By.CSS_SELECTOR, 'span.sent-timestamp')
+    _time_received_locator = (By.CSS_SELECTOR, 'span.received-timestamp')
+    _header_locator = (By.ID, 'information-report-header')
+
+    _sender = (By.CSS_SELECTOR, 'a[data-name]')
+    _receiver = (By.CSS_SELECTOR, 'a.suggestion.js-contact-info')
```

Comment:
After taking a look at the DOM, the `_sender` and the `_receiver` are actually the same element but not next to the same one. I'd use these selectors instead:
``` py
_sender = 'span[data-l10n-id="report-from-title"] ~ .js-contact-info' 
_receiver = 'span[data-l10n-id="report-to-title"] ~ .js-contact-info' 
```